[Docs] maintainers_guidelines: add WG and real-time chat request info (#79750)
ansible/ansible: docs/docsite/rst/community/maintainers_guidelines.rst

See :ref:`communication` for more details on these communication channels.

.. _wg_and_real_time_chat:

Working Group and Real-time Chat Request
----------------------------------------

> To make communication within your community more efficient and interactive, you can:

Reviewer: ```suggestion
Here are some techniques for communicating more efficiently and interactively within your community:
```

> Working Group and Real-time Chat Request
> ----------------------------------------
> 
> Here are some techniques for communicating more efficiently and interactively within your community:

Reviewer: ```suggestion
Establishing working group communication
----------------------------------------------------------------

Working groups depend on efficient, real-time communication.
Project maintainers can use the following techniques to establish communication for working groups:
```

Reviewer: How about this? The heading as-is doesn't seem to set the context quite right imo.

Author: i wasn't sure if chats is the only way they can communicate but as there's nothing else in communication guide Working Groups section (e.g. no mailing list, etc, only matrix/irc) I think it's fair to have `chat communication` in the title.
Should we put a blank line between lines 60 and 61 for readability?

Reviewer: yeah maybe "chats" isn't quite right. `s/Establishing chat communication/Establishing communication`

I put each sentence on a separate line as a force of habit. ventilated prose ftw. I don't think a blank line is really necessary.

Author: wfm, committed, PTAL

> * Join existing suitable :ref:`working_group_list` and corresponding real-time chats if exist.

Reviewer: ```suggestion
* Find an existing :ref:`working_group_list` that is similar to your project and join the conversation.
```

* `Request <https://github.com/ansible/community/blob/main/WORKING-GROUPS.md>`_ a new working group for your project.

> * `Create <https://hackmd.io/@ansible-community/community-matrix-faq#How-do-I-create-a-public-community-room>`_ a real-time chat for your working group or `ask <https://github.com/ansible/community/issues/new>`_ the community team to create it and make it official.

Reviewer: ```suggestion
* `Create <https://hackmd.io/@ansible-community/community-matrix-faq#How-do-I-create-a-public-community-room>`_ a public chat for your working group or `ask <https://github.com/ansible/community/issues/new>`_ the community team.
```

> * Add information about the working group and related real-time chats to your project's ``README.md`` and to its contributor documentation.

Reviewer: ```suggestion
* Provide working group details and links to chat rooms in the contributor section of your project ``README.md``.
```

Contributor Summits
-------------------

The quarterly Ansible Contributor Summit is a global event that provides our contributors a great opportunity to meet each other, communicate, share ideas, and see that there are other real people behind the messages on Matrix or Libera Chat IRC, or GitHub. This gives a sense of community. Watch the `Bullhorn newsletter <https://github.com/ansible/community/wiki/News#the-bullhorn>`_ for information when the next contributor summit, invite contributors you know, and take part in the event together.

Weekly community Matrix/IRC meetings
------------------------------------

> The Community and the Steering Committee come together at weekly meetings in the ``#ansible-community`` `Libera.Chat IRC <https://docs.ansible.com/ansible/devel/community/communication.html#ansible-community-on-irc>`_ channel or in the bridged `#community:ansible.com <https://matrix.to/#/#community:ansible.com>`_ room on `Matrix <https://docs.ansible.com/ansible/devel/community/communication.html#ansible-community-on-matrix>`_ to discuss important project-scale questions. See the `schedule <https://github.com/ansible/community/blob/main/meetings/README.md#schedule>`_ and join.

Reviewer: ```suggestion
The Community and the Steering Committee come together at weekly meetings in the ``#ansible-community`` `Libera.Chat IRC <https://docs.ansible.com/ansible/devel/community/communication.html#ansible-community-on-irc>`_ channel or in the bridged `#community:ansible.com <https://matrix.to/#/#community:ansible.com>`_ room on `Matrix <https://docs.ansible.com/ansible/devel/community/communication.html#ansible-community-on-matrix>`_ to discuss important project questions. Join us! Here is our `schedule <https://github.com/ansible/community/blob/main/meetings/README.md#schedule>`_.
```

Here are some ways you can expand the community around your collection:

  * Give :ref:`newcomers a positive first experience <collection_new_contributors>`.

>   * Offer them to join related :ref:`real-time chats <wg_and_real_time_chat>`.

Reviewer: ```suggestion
  * Invite contributors to join :ref:`real-time chats <wg_and_real_time_chat>` related to your project.
```
